lib/admin/class-groups-admin-user-profile.php

```diff
@@ -71,7 +71,7 @@ public static function user_new_form( $type = null ) {
 			if ( current_user_can( GROUPS_ADMINISTER_GROUPS ) ) {
 				$output = '<h3>' . _x( 'Groups', 'Groups section heading (add user)', 'groups' ) . '</h3>';
 				$groups_table = _groups_get_tablename( 'group' );
-				if ( $groups = $wpdb->get_results( "SELECT * FROM $groups_table ORDER BY name" ) ) {
+				if ( $groups = apply_filters( 'groups_add_new_user', $wpdb->get_results( "SELECT * FROM $groups_table ORDER BY name" ) ) ) {
```

Comment:
We need to review this filter because it does not adhere to our naming conventions, noted for review and inclusion after adjustment.

Reply:
Ah. My apologies. Wasn't aware of the naming convention. Thanks.